README.md

```diff
@@ -11,10 +11,38 @@
 KubeFleet is a sandbox project of the [Cloud Native Computing Foundation](https://cncf.io/) (CNCF) that works on any Kubernetes cluster.
 We are working towards the vision where we will eventually be able to treat each Kubernetes cluster as a [cattle](https://cloudscaling.com/blog/cloud-computing/the-history-of-pets-vs-cattle/).
 
+## What is KubeFleet?
+KubeFleet contains a set of Kubernetes controllers and CRDs which provide an advanced cloud-native solution for multi-cluster application management.
 
-## Code of Conduct
+Use KubeFleet to schedule workloads smartly, roll out changes progressively, and perform administrative tasks easily, across a group of Kubernetes clusters on any cloud or on-premises clusters.
+
+
+## Quickstart
+
+* [Get started here](https://kubefleet-dev.github.io/website/docs/getting-started/kind/)
+
+## Documentation
+
+To learn more about KubeFleet go to the [the KubeFleet documentation](https://kubefleet-dev.github.io/website/).
```

Comment:
Hi Ryan! I think the duplicate `the` issue re-appears somehow

Reply:
Same to a lot of the raised issues... Is there something wrong on my end?

Reply:
fixed, do you see more duplicates?